src/command_modules/azure-cli-reservations/HISTORY.rst

```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+0.2.1
+++++++
+* minor fixes regarding enums.
```

Comment:
What are the minor fixes?
Can you tell me what the new sdk changes consist of?

Reply:
I had a comment from the REST API specs review that I should use string for “location”, "provisioningState' and all ARM defined attributes. Thus, I changed those two properties from enum to string.

Reply:
https://github.com/Azure/azure-rest-api-specs/pull/3235/files

Reply:
Sounds fine, thanks